src/request/sign-transaction/SignTransaction.js

```diff
@@ -102,6 +123,10 @@ class SignTransaction {
                 this._onConfirm(request, resolve, reject, password);
             },
         );
+
+        if ('expires' in request && request.expires !== undefined) {
```

Comment:
Again, I think expires can not be `0`, so a truthy check is enough.

```suggestion
        if (request.expires) {
```

Reply:
The `'expires' in request` is required for ts to know that this is a request on which `expires` exists.